agent/config/config_unix.go

```diff
@@ -100,7 +100,7 @@ func DefaultConfig() Config {
 		PollingMetricsWaitDuration:          DefaultPollingMetricsWaitDuration,
 		NvidiaRuntime:                       DefaultNvidiaRuntime,
 		CgroupCPUPeriod:                     defaultCgroupCPUPeriod,
-		GMSACapable:                         false,
+		GMSACapable:                         isGMSACapable(),
```

Comment:
shouldn't this call `parseGMSACapability()` which is in `parse_linux.go`

Reply:
good point, made change accordingly